src/core/ColorPicker/ColorSwatch.tsx

```diff
@@ -0,0 +1,52 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import { getWindow } from '../utils/common';
+import { CommonProps } from '../utils/props';
+import cx from 'classnames';
+import '@itwin/itwinui-css/css/color-picker.css';
+
+export type ColorSwatchProps = {
+  /**
+   * Color code.
+   */
+  color: string;
+  /**
+   * Is color selected.
+   */
+  isActive?: boolean;
+  /**
+   * User friendly color name.
+   */
+  name?: string;
+} & CommonProps &
+  React.ComponentPropsWithRef<'span'>;
+
+export const ColorSwatch = React.forwardRef<HTMLSpanElement, ColorSwatchProps>(
```

Comment:
Since this component is exported, you might want to add a JSDoc comment here

Reply:
Also you need some tests for this component too

Reply:
Added comment and tests